site/search/Search.tsx

```diff
@@ -94,7 +94,8 @@ export const Search = ({
                     <SearchTopicsRefinementList topicType={topicType} />
                     <SearchResultTypeToggle />
                 </div>
-                <div className="search-template-results grid span-cols-14 grid grid-cols-12-full-width">
+                <div className="search-template-results col-start-2 span-cols-12">
```

Comment:
We don't need the grid here, and it interfered with margins for the horizontal separators.